src/main/java/org/folio/service/PatronBlocksService.java

```diff
@@ -120,6 +116,11 @@ private AutomatedPatronBlocks calculateBlocks(BlocksCalculationContext ctx) {
   private Future<BlocksCalculationContext> addAllPatronBlockConditionsToContext(
     BlocksCalculationContext ctx) {
 
+    if(ctx.patronBlockLimits.isEmpty()) {
+      ctx.logFailedValidationError("addAllPatronBlockConditionsToContext");
+      return failedFuture(DEFAULT_ERROR_MESSAGE);
```

Comment:
Why do we log 'failed validation' and return a failed future here? Absence of limits is a valid scenario, we just should skip blocks calculation and preparations for it.

Reply:
fixed